### DISCUSSION TITLE
TSR types

### DISCUSSION THREAD

**Discussion by jerronl** [aeon-toolkit/aeon#647] [Q&A] [answered]
In https://github.com/aeon-toolkit/aeon/blob/main/examples/regression/regression.ipynb, it mentioned there are two types of TSR. Can we have examples of both? Or they are already there but I didn't notice? 
Thanks!

**Comment by MatthewMiddlehurst** [comment, marked as answer]
Hi, the notebook currently contains examples for TSER (predicting a separate continuous value using time series data, similar to a classification problem).

For forecasting using supervised regressors, this notebook should help more https://github.com/aeon-toolkit/aeon/blob/main/examples/forecasting/forecasting_sklearn.ipynb.

This could be better explained in the notebook itself, I will create a PR.
